test/src/boost_test_helper.h

```diff
@@ -0,0 +1,96 @@
+//
+// Created by steven on 1/15/16.
+//
+// Replace some with with BOOST later
+//
+
+#ifndef DENOVOGEAR_BOOST_TEST_HELPER_H
+#define DENOVOGEAR_BOOST_TEST_HELPER_H
+
+#include <boost/test/test_tools.hpp>
+#include <boost/test/unit_test.hpp>
+
+//TODO: Where should this file go?
+//FIXME: too many global
+const double ABS_TEST_THRESHOLD = 1e-10;
+const double BOOST_CLOSE_THRESHOLD = 1e-3;//
+const double BOOST_SMALL_THRESHOLD = sqrt(std::numeric_limits<double>::epsilon());
+//TODO: What is a good cut? 1e-8?// sqrt(std::numeric_limits<double>::epsilon());
+
```

Comment:
I think we should make this configurable somehow from a `CMake` run.  It is important to test both absolute equality and threshold-based equality.